src/test/kotlin/open/tresorier/services/OperationServiceTest.kt

```diff
@@ -57,10 +57,10 @@ class OperationServiceTest : ITest {
         masterCategoryDao.insert(masterCategory)
         val category = Category("oftenAllocatedCategory", masterCategory.id)
         categoryDao.insert(category)
-        val operationCreated = operationService.create(emilie, account, TestData.jan_16_2022, category, 1000, "", false)
+        val operationCreated = operationService.create(emilie, account, TestData.jan_16_2022, category, 1000, "", false, null)
```

Comment:
since the param is nullable, you shouldn't need to add null for each call

Reply:
it need parameter null otherwise it doesn't pass